packages/fftw3/fftw3.0.8.5/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+authors: ["Christophe Troestler <Christophe.Troestler@umons.ac.be>"]
+maintainer: "Christophe.Troestler@umons.ac.be"
+license: "LGPL-2.1"
+homepage: "https://github.com/Chris00/fftw-ocaml"
+dev-repo: "git+https://github.com/Chris00/fftw-ocaml.git"
+bug-reports: "https://github.com/Chris00/fftw-ocaml/issues"
+doc: "https://Chris00.github.io/fftw-ocaml/doc"
+tags: ["FFT"]
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.1"}
+  "dune-configurator"
```

Comment:
```suggestion
  "dune-configurator" {>= "1.11.4"}
```
I think it changed name there. Let's see if this fixes it, or it happened in 2.0

Reply:
🎉